lib/matplotlib/collections.py

```diff
@@ -229,6 +227,11 @@ def get_offset_transform(self):
             t = t._as_mpl_transform(self.axes)
         return t
 
+    def set_offset_transform(self, transOffset):
+        self._transOffset = transOffset
```

Comment:
If `transOffset is None` in `__init__`, then it would remain with the class default `IdentifyTransform`, so should this not check for `None` and do the same?

Reply:
I copied the behaviour of `Artist.set_transform`, though the initialization/getter behaviour is different, so I agree that it should change. `Artist._transform` initializes to `None`, and sets the value to `IdentityTransform()` when `get_transform()` is called. I'll change the getter/initialization instead to keep the same behaviour for those too as well.

Reply:
I've updated `get_offset_transform` and changed `__init__` slightly, I hope it looks better now. Thanks for your feedback!